content/docs/user-guide/assets/asset-processor/settings.md

```diff
@@ -8,34 +8,39 @@ toc: true
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| acceptInput                     | Enable external control messaging via the ControlRequestHandler, used with automated tests.                                                                                                                 |                                                                                                 |
+| additionalScanFolders           | Used with `dependencyScanPattern` to farther filter the scan.                                                                                                                                               |                                                                                                 |
+| debugOutput                     | When enabled, builders that support it will output debug information as product assets. Used primarily with scene files.                                                                                    |                                                                                                 |
+| dependencyScanMaxIteration      | Used to limit the number of recursive searches per line when running dependencyScanPattern.                                                                                                                 | Default is 800.                                                                                 |
+| dependencyScanPattern (dsp)     | Scans assets that match the given pattern for missing product dependencies.                                                                                                                                 |                                                                                                 |
+| enableQueryLogging              | Enables logging database queries.                                                                                                                                                                           |                                                                                                 |
+| fileDependencyScanPattern (fsp) | Used with `dependencyScanPattern` to farther filter the scan.                                                                                                                                               |                                                                                                 |
```

Comment:
I think this should be "further" instead of "farther"

Reply:
Fixed. Copied typo from code comment. Nice catch